modules/tinymce/src/plugins/table/test/ts/browser/ui/ColorPickerTest.ts

```diff
@@ -17,7 +17,7 @@
 // import SilverTheme from '../../../../../themes/silver/main/ts/Theme';
 import { UnitTest } from '@ephox/bedrock-client';
 
-UnitTest.asynctest('browser.tinymce.plugins.table.ColorPickerTest', (success, failure) => {
+UnitTest.asynctest('browser.tinymce.plugins.table.ColorPickerTest', (success, _failure) => {
```

Comment:
Should we just delete this file instead, as it's not doing anything and is completely commented out?

Reply:
If this test is needed, we should probably log a Jira to get it working again 🤷‍♀

Reply:
I tried to follow the history to find out why it was commented out and it seems to have always been commented out? Either that or git-lense is glitching which I can't rule out since it seems to break when I view the older commits.

Reply:
Yeah, it's commented out in v4 as well: https://github.com/tinymce/tinymce/blob/4.x/src/plugins/table/test/ts/browser/ui/ColorPickerTest.ts so I'd opt to just delete the file.

Reply:
I have deleted it. I suppose we can always revive it from history if it turns out important (assuming we remember it existed).